templates/layout.html

```diff
@@ -1,35 +1,90 @@
 <!DOCTYPE html>
```

Comment:
So usually from what I've seen the top-most template in the hierarchy tends to be called `base.html`. Also you might want to consider moving the navbar stuff to a separate template that gets included. I like my `base.html` to be nothing but includes and the basic html skeleton of the site.